src/main/java/org/sonarlint/cli/SonarLint.java

```diff
@@ -47,8 +44,15 @@
 public class SonarLint {
   private boolean running;
   private SonarLintClient client;
+  private Logger logger;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make the "logger" logger private static final and rename it to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1312)